twitch.cpp

```diff
@@ -67,9 +71,24 @@ std::set<std::string> Twitch::getOnlineUsers(const std::string& channel) {
         }
     }
 
+    auto current_time = std::chrono::system_clock::now();
+    auto user_snapshot = _recent_chatters;
+    _recent_chatters = {};
+    for(auto const& [user, last_heard_from] : user_snapshot) {
+        if (current_time - last_heard_from < std::chrono::minutes(5)) {
```

Comment:
can you add a TODO that this should be runtime-configurable

Reply:
Done